uitestutils/src/main/java/com/gravatar/uitestutils/RoborazziTest.kt

```diff
@@ -43,6 +43,7 @@ abstract class RoborazziTest {
             roborazziOptions = RoborazziOptions(
                 compareOptions = RoborazziOptions.CompareOptions(
                     imageComparator = SimpleImageComparator(maxDistance = 0.007F, hShift = 1),
+                    changeThreshold = 0.01f,
```

Comment:
Not happy about that, but it's getting quite annoying to have tests fail on CI because of system differences. I made some changes some time ago that fixed the majority of issues, but now there's one test that fails `AvatarPickerListLoaded` - the dark version passes...

I think this will work for now and when time allows we should modify the CI flow to generate the baseline as per [this](https://github.com/Automattic/Gravatar-SDK-Android/pull/328).

Reply:
[yes, this is the correct one.](https://github.com/takahirom/roborazzi/?tab=readme-ov-file#q-why-do-my-screenshot-tests-fail-inconsistently-across-different-operating-systems-like-macos-ubuntu-and-windows)

Reply:
https://github.com/Automattic/Gravatar-SDK-Android/issues/333